TDD_Training/TDD_Training/FizzBuzzService.cs

```diff
@@ -6,27 +6,52 @@ public class FizzBuzzService
     {
         public string CheckFizzBuzz(int givenNumber)
         {
-            switch(givenNumber)
+            try
```

Comment:
No need for try-catch in this service, since you are already throwing the exception.